### PR TITLE
Fixes config for previews

### PR DESCRIPTION
**Pull request by paul-vd**

### CODE REVIEW

File: source/docs/prismic/preview.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ In your repository, go to Settings > Previews > Manage your Previews and select
 
 - **Site Name:** The display name to identify the website in the preview.
 - **Domain for Your Application:** The URL of your site, such as https://example.com/ or http://localhost:3000/
-- **Link Resolver:** A route in your app that provides a Link Resolver. Some of Prismic's app development kits create this route for you at /preview.
+- **Link Resolver:** The route where prismic resolves the preview url, in Front-Commerce the route is resolved at `/prismic/previews`.
```
Comment (mo-fc) [inline source/docs/prismic/preview.md:16]:
```suggestion
- **Link Resolver:** The route where prismic resolves the preview url. In Front-Commerce the route is resolved at `/prismic/previews`.
```

File: source/docs/prismic/preview.md [hunk 2]
```diff
@@ -13,10 +13,10 @@ In your repository, go to Settings > Previews > Manage your Previews and select
...
 
 <div style="text-align:center;">
-![Screenshot of a repository's preview configuration](./assets/previews/create-new-preview.jpg)
+  <img src="./assets/previews/create-new-preview.jpg" alt="Screenshot of a repository's preview configuration" style="border-radius:5px;">
```
Comment (mo-fc) [inline source/docs/prismic/preview.md:19]:
```suggestion
  <img src="./assets/previews/create-new-preview.jpg" alt="Screenshot of a repository's preview configuration">
```

**Review by mo-fc — CHANGES_REQUESTED** [review]

File: source/docs/prismic/preview.md (hunk 2 above)
Comment (paul-vd) [reply source/docs/prismic/preview.md:19]: I did this on purpose to have the same border that the code lines, looks much better :)

Comment (paul-vd) [reply source/docs/prismic/preview.md:19]: image

**Review by real34 — APPROVED** [review]

**Pull request merged**